### PR TITLE
Added checks in CI environment

### PR DESCRIPTION
**Pull request by ARMmaster17**
This PR contains the following changes:
- Brakeman security checks on CI builds.
- RuboCop checks on CI builds.
- Updated Gemfile.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -140,14 +287,15 @@ DEPENDENCIES
...
   sinatra-activerecord
   slim
 
 RUBY VERSION
-   ruby 2.3.0p0
+   ruby 2.4.0p0
 
 BUNDLED WITH
-   1.13.6
+   1.14.3
```
Comment (ARMmaster17) [inline Gemfile.lock]: This is the part that may potentially break. The Heroku docs specify that every Ruby project is built with Bundler 1.13.7. 1.13.6 refused to install dependencies because of an issue with `pry`.

Comment (ARMmaster17) [inline Gemfile.lock:301]: This is the part that may potentially break. The Heroku docs specify that every Ruby project is built with Bundler 1.13.7. 1.13.6 refused to install dependencies because of an issue with `pry`.

File: Gemfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.3.0'
+ruby '2.4.0'
```
Comment (ARMmaster17) [inline Gemfile:3]: This part may also break some stuff. The unit tests showed up clean, but there may be some edge cases that can't be tested without the production environment.

**Review by wulf502 — APPROVED** [review]
